op-guide/binary-deployment.md

```diff
@@ -132,7 +132,9 @@ The key's randomart image is:
 
 TiDB provides the official binary installation package that supports Linux. For the operating system, it is recommended to use Redhat 7.3+, CentOS 7.3+ and higher versions.
 
-### Operating system: Linux (Redhat 7+, CentOS 7+)
+> Note: Because TiDB is compatible with MySQL, you can use MySQL client to connect to TiDB directly. Before you start TiDB, make sure to install MySQL client first.
```

Comment:
Installing before starting TiDB is not necessary. I think the last sentence can be removed.